### DISCUSSION TITLE
2023-09-19

### DISCUSSION THREAD

**Discussion by aak2075** [woowacourse-teams/2023-haru-study#529] [daily_meeting]
### 데일리 미팅

개인 일정
- 노아 : 5시 근로
- 백 & 프론트 : 오전 강의

### 오늘의 할 일

- 프론트엔드 : 이미지 최적화
- 백엔드 : 인덱스, 성능 개선

### 내일의 할 일

- 배포 : 랜딩 페이지 변경, 반응형, 에러 출력, 모달, 페이지 이동

**Comment by jaehee329** [comment]
### 체크인
6점 - 여기 왤케 춥죠?
### 체크아웃
6점 - 부하테스트 결과 해석이 어렵네요

**Comment by nlom0218** [comment]
### 체크인

9점 - 캠퍼스 너무 시원해요

### 체크아웃

6점 - 왜? 오후 이후엔 에어컨을 끄는거죠? 제일 더울 때가 오후인데

**Comment by woosung1223** [comment]
### 체크인
8점 - 강의 들을 생각에 좋네요

### 체크아웃
7점 - 왜 커넥션 큐가 이론처럼 동작하지 않는걸까요

**Comment by aak2075** [comment]
### 체크인

10점 - 추웠는데 스윗 노아가 옷을 줬어요

### 체크아웃

10점 - 부하테스트 재밌네요

**Comment by MoonJeWoong** [comment]
### 체크인
8점 - 강의... 미션... 그리고 최소 요구사항... 화이팅...


### 체크아웃
5점 - 쓰레드 풀... 헤엄...헤엄...헤엄....

**Comment by yeopto** [comment]
### 체크인

10점 - 어제 농구로 땀 빼고 숙면을 취해서 좋습니다.

### 체크아웃

10점 - 반응형 끝내서 기분이 좋습니다.

**Comment by woo-jk** [comment]
### 체크인
8점 - 오늘 하루 화이팅~

### 체크아웃
9점 - 피곤하지만 아침보단 낫다
